index.html

```diff
@@ -1,71 +1,121 @@
+        </div>
+        <div class="paid item-list">
+            <div class="lh">
+                TOTAL PAID
+            </div>
+            <div class="container">
+                <ul>
+                    <li>75 <a href="https://youtu.be/YixWlAtvNhk?t=16440">community subs</a> ($375)</li>
+                    <li>272,310 <a href="https://youtu.be/USwGGyfczR8">lego studs</a> (1 USD = 1 LS) = $272,310</li>
+                    <li>$100 as an <a href="https://youtu.be/MhycXVLPn7I?t=463">apology for calling Jerma a "boring man"
+                            earlier</a></li>
+                    <li>$100,000,000 <a href="https://youtu.be/ZNqtvglRdcE?t=20014">SHOWED OTTO</a> :)</li>
+                    <li>$153,000 in <a href="https://youtu.be/QIuS5aLu1_M?t=19368">pizza sales</a></li>
+                    <li>$30,715.67 for <a href="https://youtu.be/QIuS5aLu1_M?t=19368">¥3,204,259 Yen in Yakuza 0</a> (1
+                        USD
+                        = 104.32 JPY, <a href="https://www.exchange-rates.org/Rate/USD/JPY/12-1-2020">exchange rate on
+                            December 1st 2020</a>)</li>
+                    <li>$250,000,000 for a <a href="https://youtu.be/a9uuyksatqE?t=9996">flawless reading</a> of green
+                        eggs
+                        and ham
+                </ul>
+            </div>
+        </div>
+        <div class="added item-list">
+            <div class="lh">
+                DEBT ADDED
+            </div>
+            <div class="container">
+                <ul>
+                    <li>$2,399,727,315 the worst <a href="https://youtu.be/BQaSlD_UFRU?t=10675">Deal or No Deal</a> play
+                        of
+                        all time</li>
+                    <li>$250,000,000 <a href="https://youtu.be/BQaSlD_UFRU?t=7300">Jerma bets 250 million that he hasn't
+                            said "GAS" more than the movie's script.</a> He failed the bet.</li>
+                    <li><del>$1,000,000,000 <a href="https://youtu.be/5RL0ZY2xpZs?t=6431">Jerma bets $1 billion that he
+                                will
+                                finish The Impossible Quiz.</a> He failed the bet.</del> Added in error. <a
+                            href="https://www.reddit.com/r/jerma985/comments/kfw0gd/audit_the_debt/">Thanks for
+                            auditing!</a> ;)</li>
+                    <li><del>$50,000,000,000 <a href="https://clips.twitch.tv/SweetSecretiveLardPermaSmug">bad self dare
+                                and
+                                bad hitbox</a></del> Reversed: Polls decided (<a
+                            href="https://www.strawpoll.me/42283739/r">[1]</a> <a
+                            href="https://www.strawpoll.me/42283673/r">[2]</a> <a
+                            href="https://www.strawpoll.me/42283794/r">[3]</a>)</li>
+                    <li> For invalidated/removed debts, check <a href="https://debt.jerma.io/invalidated">here.</a></li>
```

Comment:
same issue. stuff should not be on new lines unless it actually has a line break

Reply:
Industry Standard.